Description: Use pkg-config to determine FFmpeg linker flags

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Last-Update: <2014-07-27>

--- acoustid-fingerprinter-0.6.orig/CMakeLists.txt
+++ acoustid-fingerprinter-0.6/CMakeLists.txt
@@ -12,6 +12,7 @@ set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
 
 find_package(Qt4 COMPONENTS QtCore QtGui QtNetwork REQUIRED)
+find_package(PkgConfig REQUIRED)
 find_package(FFmpeg REQUIRED)
 find_package(Taglib REQUIRED)
 find_package(Chromaprint REQUIRED)
--- acoustid-fingerprinter-0.6.orig/cmake/modules/FindFFmpeg.cmake
+++ acoustid-fingerprinter-0.6/cmake/modules/FindFFmpeg.cmake
@@ -47,29 +47,7 @@ MACRO(FFMPEG_FIND varname shortname head
 		DOC "Location of FFMPEG Headers"
 	)
 
-    FIND_LIBRARY(FFMPEG_${varname}_LIBRARIES
-        NAMES ${shortname}
-        PATHS
-        ${FFMPEG_ROOT}/lib
-        $ENV{FFMPEG_DIR}/lib
-        ~/Library/Frameworks
-        /Library/Frameworks
-        /usr/local/lib
-        /usr/local/lib64
-        /usr/lib
-        /usr/lib64
-        /sw/lib
-        /opt/local/lib
-        /opt/csw/lib
-        /opt/lib
-        /usr/freeware/lib64
-		NO_DEFAULT_PATH
-        DOC "Location of FFMPEG Libraries"
-    )
-    FIND_LIBRARY(FFMPEG_${varname}_LIBRARIES
-        NAMES ${shortname}
-        DOC "Location of FFMPEG Libraries"
-    )
+	pkg_check_modules(FFMPEG_${varname} lib${shortname})
 
     IF (FFMPEG_${varname}_LIBRARIES AND FFMPEG_${varname}_INCLUDE_DIRS)
         SET(FFMPEG_${varname}_FOUND 1)
