From 6cb95c67cd9699fb3d703451eb1c4bcabc96e25f Mon Sep 17 00:00:00 2001
From: Wieland Hoffmann <themineo+bb@gmail.com>
Date: Tue, 5 Nov 2013 12:08:52 +0000
Subject: [PATCH] Change 'Submittion' to 'Submission' in a warning message

---
 fingerprinter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fingerprinter.cpp b/fingerprinter.cpp
index db03313..940a70b 100644
--- a/fingerprinter.cpp
+++ b/fingerprinter.cpp
@@ -261,7 +261,7 @@ void Fingerprinter::onRequestFinished(QNetworkReply *reply)
 			stop = true;
 		}
 		else {
-			qWarning() << "Submittion failed:" << errorMessage;
+			qWarning() << "Submission failed:" << errorMessage;
 		}
 	}
 	else if (error != QNetworkReply::NoError) {
